src/cascadia/LocalTests_SettingsModel/DeserializationTests.cpp

```diff
@@ -1423,6 +1424,28 @@ namespace SettingsModelLocalTests
         VERIFY_ARE_NOT_EQUAL(0u, settings->_profiles.Size());
         VERIFY_ARE_EQUAL(expectedPath, settings->_profiles.GetAt(0).ExpandedBackgroundImagePath());
     }
+    void DeserializationTests::TestProfileBackgroundImageWithDesktopWallpaper()

```

Comment:
This test looks good. Thanks!

If you want to run the/a test, we use TAEF (read more [here](https://github.com/microsoft/terminal/blob/master/doc/TAEF.md)). (I should probably update that doc with more info or create a new one). Do the following steps:
- build `LocalTests_SettingsModel`
- _rebuild_ `TestHostApp` (sometimes doing a plain build won't copy the LocalTest DLL)
- in a command line, go to `x64\Debug\TestHostApp`
- `TE.exe /name:<name_of_test> SettingsModel.LocalTests.dll`
   - replace <name_of_test> with a regex (i.e `*DeserializationTests* to run all of these, *TestProfileBackground* for this new one)
   - TE.exe may not be in this folder, but you can invoke one from another location. I ran it from `..\..\..\bin\x64\Debug\LocalTests_TerminalApp\TE.exe`

_Generally_, we run tests in CI (except local tests), but we've been having a few issues lately for some reason.

Reply:
The reason this doesn't error is because when we try, we pass in the ExpandedBackgroundImagePath, which is a valid image path if "DesktopWallpaper". I manually confirmed if you pass in "Hello" or some other invalid option, it will state it as an invalid.

Reply:
> You didn't have to update `_ValidateMediaResources`? I expected that when you set the background image to `DesktopWallpaper`, Terminal would give you a popup warning that it's not valid.
> 
> And don't worry about "slowing down", happy to help anyway I can. And hey, the more you learn, the more you can jump into the code! :) Thanks for the help!

The reason this doesn't error is because when we try, we pass in the ExpandedBackgroundImagePath, which is a valid image path if "DesktopWallpaper". I manually confirmed if you pass in "Hello" or some other invalid option, it will state it as an invalid.

Reply:
![image](https://user-images.githubusercontent.com/46642544/95497550-bc434380-095f-11eb-95a8-fa312dc0a90a.png)